<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/elements/home.jpg);</string>
  </property>
  <widget class="QPushButton" name="rankButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>141</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/rank.png);
background: transparent;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="battleButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>512</width>
     <height>512</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/battle.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bagButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/bag.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bagLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 24pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>背包</string>
   </property>
  </widget>
  <widget class="QLabel" name="rankLable">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 24pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>排名</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 48pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>决斗！</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonCntMedalFrame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/frame2.png);
font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;宠物个数徽章&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="highPokemonMedalFrame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>660</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/frame2.png);
font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;高级宠物徽章&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonCntMedalGoldenIcon">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/golden.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonCntMedalGoldenLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>610</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>x0</string>
   </property>
  </widget>
  <widget class="QLabel" name="highPokemonMedalGoldenIcon">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/golden.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="highPokemonMedalGoldenLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>680</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>x0</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonCntMedalSilverIcon">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>590</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/silver.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="highPokemonMedalSilverLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>680</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>x0</string>
   </property>
  </widget>
  <widget class="QLabel" name="highPokemonMedalSilverIcon">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/silver.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonCntMedalSilverLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>x0</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonCntMedalBronzeIcon">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>590</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/bronze.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="highPokemonMedalBronzeLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>680</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>x0</string>
   </property>
  </widget>
  <widget class="QLabel" name="highPokemonMedalBronzeIcon">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/bronze.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonCntMedalBronzeLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>610</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>x0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>0</y>
     <width>100</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/close.png);
background: transparent;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameFrame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>771</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/name.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 48pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;训练师&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleCntLable">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>400</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/frame2.png);
font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;决斗总场次：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleWinCntLabel">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>480</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/frame2.png);
font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;决斗胜场：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleWinRateLabel">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>560</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/frame2.png);
font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;决斗胜率：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleCnt">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleWinCnt">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="battleWinRate">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>570</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Hannotate SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flashButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>640</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/flash.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
